spec/frame_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative '../lib/frame'
+
+describe Frame do
+    it 'allows a player to input each of their rolls' do
+        subject.input_roll(9)
```

Comment:
I think you could take out this test in your refactor step as it is covered by implication in future tests